docs/tutorials/control/flutter-app.md

```diff
@@ -0,0 +1,732 @@
+
+   ```yaml {class="line-numbers linkable-line-numbers"}
+   name: smart_machine_app
+   description: "A new Flutter project."
+
+   publish_to: "none" # Remove this line if you wish to publish to pub.dev
+
+   version: 1.0.0+1
+
+   environment:
+     sdk: ">=3.2.3 <4.0.0"
+
+   dependencies:
+     flutter:
+       sdk: flutter
+     flutter_dotenv: ^5.1.0
+     image: ^4.0.17
+     cupertino_icons: ^1.0.2
+     viam_sdk: ^0.0.9
+
+   dev_dependencies:
+     flutter_test:
+       sdk: flutter
+
+     flutter_lints: ^2.0.0
+
+   flutter:
+     uses-material-design: true
+   ```
+
+   {{% alert title="Note" color="note" %}}
+
+   If you named your app something other than `smart_machine_app`, you need to change the `name` value in the first line of the <file>pubspec.yaml</file> file to the name you gave your app during setup.
+
+   {{% /alert %}}
+
+   3. Next, open another configuration file in the project, <file>analysis_options.yaml</file>.
+      This file determines how strict Flutter should be when analyzing your code.
+      Since this is your first foray into Flutter, you're telling the analyzer to take it easy.
+      You can always tune this later.
+      In fact, as you get closer to publishing an actual production app, you will almost certainly want to make the analyzer stricter than this.
+
+   4. Replace its contents with the following:
+
+   ```yaml {class="line-numbers linkable-line-numbers"}
+   include: package:flutter_lints/flutter.yaml
+
+   linter:
+     rules:
+       avoid_print: false
+       prefer_const_constructors_in_immutables: false
+       prefer_const_constructors: false
+       prefer_const_literals_to_create_immutables: false
+       prefer_final_fields: false
+       unnecessary_breaks: true
+       use_key_in_widget_constructors: false
+   ```
+
+### Configure iOS-specific code
+
+Now you'll update some configurations in the iOS-specific code to support the [Viam Flutter SDK](https://flutter.viam.dev/).
+
+1. Open <file>ios/Podfile</file>.
+   If Podfile does not exist in that directory, you can generate it by running `flutter pub get` in the root directory of your app.
+
+   At the top of the file you will see the following lines:
+
+   ```{class="line-numbers linkable-line-numbers"}
+   # Uncomment this line to define a global platform for your project
+   # platform :ios, '11.0'
+   ```
+
+   Change that to this:
+
+   ```{class="line-numbers linkable-line-numbers"}
+   # Uncomment this line to define a global platform for your project
+   platform :ios, '13.0'
+   ```
+
+2. Open <file>ios/Runner/Info.plist</file>.
+   It will look something like this:
+
+   {{<imgproc src="/tutorials/flutter-app/info-plist.png" resize="1200x" style="max-width:900px" declared-dimensions="true" alt="The info.plist file open in VS Code. The second line includes 'doctype plist public' and an Apple URL. The fourth line is a dict tag. The fifth line is a key, and subsequent lines contain keys, strings and arrays.">}}
```

Comment:
This image appears to _underlay_ the right-TOC making it transparent on smaller displays (happen to be reviewing on 13" laptop atm)! Woah, never seen that before! Tricky here because we need zoom for tiny text. Do you think we could crop this to only show the file? Ah, I really like the "full IDE view" though too. I'm not sure what to do about this.

Reply:
Is it time? https://viam.atlassian.net/browse/DOCS-1584